app/views/alma_bib_norm_mailer/error_notificaiton.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>PulBibNorm POST failed</h1>
```

Comment:
Typo in file name (`error_notification` rather than `error_notificaiton`), needs newline at end of file.